robot/robot.go

```diff
@@ -110,6 +111,9 @@ type LocalRobot interface {
 	// ModuleAddress returns the address (path) of the unix socket modules use to contact the parent.
 	ModuleAddress() (string, error)
 
+	// GetCloudMetadata returns app-related information about the robot.
```

Comment:
why not put this into the Robot interface? since the client is implementing it as well

Reply:
ugh, because I got lost :disappointed: - meant to put it there, not `LocalRobot`